source/spawn/spawn-processes.ads

```diff
@@ -150,6 +150,22 @@ package Spawn.Processes is
    --  Executables name. Note that this should be a resolved path,
    --  this API does not look for executables on the PATH.
 
+   procedure Set_Standard_Input_PTY (Self : in out Process'Class);
```

Comment:
shall we raise "unimplemented" exception on platforms where don't implement it?

Reply:
No exceptions :)